cli/cmd/stat.go

```diff
@@ -53,16 +53,19 @@ func newCmdStat() *cobra.Command {
   Examples:
   * deploy
   * deploy/my-deploy
-  * deploy my-deploy
-  * ns/my-ns
+  * rc/my-replication-controller
+	* ns/my-ns
+	* authority
+	* au/my-authority
   * all
 
 Valid resource types include:
 
   * deployments
   * namespaces
   * pods
-  * replicationcontrollers
+	* replicationcontrollers
+	* authority (not supported in --from)
```

Comment:
Same comment here -- spaces instead of tabs.

Reply:
Also, I'd prefer if we use "authorities" (plural) on line 68, for consistent with the rest of the resources in the list.